src/Git/GitStatusListCtrl.cpp

```diff
@@ -4490,3 +4531,78 @@ ULONG CGitStatusListCtrl::GetGestureStatus(CPoint /*ptTouch*/)
 {
 	return 0;
 }
+
+void CGitStatusListCtrl::LoadChangelists()
+{
+	CString changelistFLP = g_Git.m_CurrentDir + L"\\.tgitchangelist";
+	if (!PathFileExists(changelistFLP))
+		return;
+
+	try
+	{
+		CStdioFile file(changelistFLP, CFile::typeText | CFile::modeRead | CFile::shareDenyWrite);
+		CString changelistName = GITSLC_IGNORECHANGELIST;
+		CString strLine;
+		while (file.ReadString(strLine))
+		{
+			strLine = strLine.Trim();
+			if (strLine.IsEmpty())
+				continue;
+
+			if ((strLine.Left(1) == L'<') && (strLine.Right(1) == L'>'))
+			{ //this is changelist name
+				changelistName = strLine.Mid(1, strLine.GetLength() - 2);
+				m_changelists.insert_or_assign(changelistName, 0);
+			}
+			else // this is git path
+				m_pathToChangelist.insert_or_assign(strLine, changelistName);
+		}
+		file.Close();
+	}
+	catch (CFileException* pE)
+	{
+		CTraceToOutputDebugString::Instance()(__FUNCTION__ ": CFileException loading changelists file\n");

```

Comment:
As we also show a messagebox, so this debug output should not be necessary as it adds no further info